weepickle-implicits/src/main/scala-3/com/rallyhealth/weepickle/v1/implicits/CaseClassFrom.scala

```diff
@@ -108,11 +109,28 @@ trait CaseClassFromPiece extends MacrosCommon:
       From.merge[T](writers:_*)
   }
 
+  /*
+   * Default string encoding for Scala 3 enums. Mirror should always be for the sealed trait generated by the
+   * enum, which is a Sum type. So any calls here with Product type mirrors are rejected as compilation errors.
+   */
+  inline def macroEnumFrom[T: ClassTag](using m: Mirror.Of[T]): From[T] = inline m match {
```

Comment:
If this is only used internally, can it be made private?

Reply:
It is called from `MacroImplicits` in the same package, so I can make it package private. Same with `macroEnumTo` in `CaseClassTo`.

Reply:
Sounds good. Wiggle room for the future.